testing/README.md

```diff
@@ -278,6 +278,306 @@ Clears all registered benchmarks, so calling `runBenchmarks()` after it wont run
 them. Filtering can be applied by setting `BenchmarkRunOptions.only` and/or
 `BenchmarkRunOptions.skip` to regular expressions matching benchmark names.
 
+## Behavior-driven development
+
+With the `bdd.ts` module you can write your tests in a familiar format for
+grouping tests and adding setup/teardown hooks used by other JavaScript testing
+frameworks like Jasmine, Jest, and Mocha.
+
+The `describe` function creates a block that groups together several related
+tests. The `it` function registers an individual test case. The `describe` and
+`it` functions have similar call signatures to `Deno.test`, making it easy to
+migrate from using `Deno.test`.
+
+### Hooks
+
+There are 4 types of hooks available for test suites. A test suite can have
+multiples of each type of hook, they will be called in the order that they are
+registered. The `afterEach` and `afterAll` hooks will be called whether or not
+the test case passes. The all hooks will be called once for the whole group
```

Comment:
maybe we should write `'All' hooks` instead of `all hooks` to make it clear what are referenced

Reply:
`*All` hooks